op-e2e/actions/l2_runtime.go

```diff
@@ -272,31 +366,23 @@ func (rt *Runtime) setupChallenge(challenger *L2Validator) {
 	require.NoError(rt.t, err)
 	require.NotNil(rt.t, challenge, "challenge not found")
 
-	// check pending bond amount after create challenge
-	pendingBond, err := rt.valPoolContract.GetPendingBond(nil, rt.outputIndex, challenger.address)
-	require.NoError(rt.t, err)
-	require.Equal(rt.t, pendingBond, rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt())
-
-	// check challenger balance decreased
-	cBal, err := rt.valPoolContract.BalanceOf(nil, challenger.address)
-	require.NoError(rt.t, err)
-	require.Equal(rt.t, new(big.Int).Sub(new(big.Int).SetInt64(defaultDepositAmount), rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt()), cBal)
-}
-
-// IsCreationEnded checks if the creation period of rt.outputIndex output is ended
-func (rt *Runtime) IsCreationEnded() bool {
-	output, err := rt.outputOracleContract.GetL2Output(nil, rt.outputIndex)
-	require.NoError(rt.t, err)
+	if version == defaultValidatorSystemVersion {
+		// check pending bond amount after create challenge
+		pendingBond, err := rt.valPoolContract.GetPendingBond(nil, rt.outputIndex, challenger.address)
+		require.NoError(rt.t, err)
+		require.Equal(rt.t, pendingBond, rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt())
 
-	ended := output.Timestamp.Uint64() + rt.dp.DeployConfig.ColosseumCreationPeriodSeconds
-	isEnded := rt.miner.l1Chain.CurrentBlock().Time > ended
-	return isEnded
+		// check challenger balance decreased
+		cBal, err := rt.valPoolContract.BalanceOf(nil, challenger.address)
+		require.NoError(rt.t, err)
+		require.Equal(rt.t, new(big.Int).Sub(new(big.Int).SetInt64(defaultDepositAmount), rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt()), cBal)
+	}
 }
 
-func (rt *Runtime) SetCreationPeriod(period uint64) {
-	rt.dp.DeployConfig.ColosseumCreationPeriodSeconds = period
+func (rt *Runtime) includeL1BlockBySender(from common.Address) {
+	rt.miner.includeL1BlockBySender(rt.t, from, rt.l1BlockDelta)
 }
```

Comment:
Removed `IsCreationEnded` and `SetCreationPeriod` since now we cannot change deploy config in this way.